server/controllers/notifications.js

```diff
@@ -0,0 +1,77 @@
+const Notification = require("../models/Notification");
+const asyncHandler = require("express-async-handler");
+
+//get all notifications
+exports.getAllNotifications = asyncHandler(async (req, res, next) => {
+let notifications = await Notification.find()
+
+if(!notifications){
+    res.status(404)
+    throw new Error('No notifications found!')
+}
+// can we shorten it this way since the key and value are the same?
+res.status(200).json({notifications})
+});
+
+exports.getAllUnreadNotifications = asyncHandler(async (req, res, next) => {
+    // feel like this is wrong
+let unreadNotifications = await Notification.find({read: false})
```

Comment:
Both this and the above don't really have a use case, can you change these routes to fetch all notifications and unread notifications for the logged in user? Right now it's fetching all for all users